src/pages/console/authz/permissions.mdx

```diff
@@ -16,26 +16,26 @@ You are going to define set up role-based authorization rules for each of the mo
 2. In the **Model** pane on the right, expand the **Owners** window. 
 3. Choose **Create**, **Read**, **Update** and **Delete** to specify that *Owners* have create, read, update, and delete access. The settings look as follows.
 
-![GSA](/images/console/10_ownersaccess.png)
+![Owner authorization rules on a GraphQL data model](/images/console/10_ownersaccess.png)
```

Comment:
Does this image show extra settings someone would have to configure that isn't already mentioned above? I see that we're supposed to have create, read, update, and delete but am wondering if there's anything else that is being shown in the pic.

If it does, then we'll need to write that out inside the alt text, or we explain it in more detail in the paragraph above.

The reason I bring this up is because we mention, "The settings look as follows...". Someone using a screen reader might not be able to get that extra info from the picture if it isn't mentioned anywhere else.